docs/source/devel.rst

```diff
@@ -60,6 +60,35 @@ to control the order in which the suites and test-cases are executed.
 
 As a general rule, we should avoid using words like "check" or "verify" in suite names.
 
+Although slightly dated, the original naming convention authored by Robot
+Framework project still applies to this project:
+
+ref: `Test suite names`: https://github.com/robotframework/HowToWriteGoodTestCases/blob/master/HowToWriteGoodTestCases.rst#test-suite-names
+
+Test-case names
+~~~~~~~~~~~~~~~
+
+Test case names should describe what a test does and preferably begins with a
+verb to indicate the action being tested.
+
+In our project we use sentence case for test case names: where the first character
+for the name is upper-case, and the rest of the name is a sentence. Eg:
+
+"One client writes a file and unauthorized user cannot append the file".
```

Comment:
It's nice to have an example, but it would be helpful to specify a convention.  

Something to make the names consistent.  What is the general structure of the name? Should it describe the "behaivor"?  In the sentence above, I'm not sure what is the given, and what is the change, and what is expected.  What does "one client" have to do with "authorized user"?